### PR TITLE
[ja] fix a nit in the feature-state short code

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/32474/files
/assign @atoato88
/assign @nasa9084

### CODE REVIEW

File: content/ja/docs/concepts/workloads/controllers/daemonset.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ selector](/ja/docs/concepts/scheduling-eviction/assign-pod-node/)にマッチす
 
 ### デフォルトスケジューラーによってスケジューリングされる場合
 
-{{< feature-state state="stable" for-kubernetes-version="1.17" >}}
+{{< feature-state for_kubernetes_version="1.17" state="stable" >}}
```
Comment (atoato88) [inline content/ja/docs/concepts/workloads/controllers/daemonset.md:83]:
```suggestion
{{< feature-state for_k8s_version="1.17" state="stable" >}}
```
Is valid property name `for_k8s_version`, not `for_kubernetes_version`?

I could found `for_k8s_version` around other doc.
e.g. [here](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/controllers/job.md#completion-mode)

**Review by atoato88 — COMMENTED** [review]
[Document on preview site](https://deploy-preview-33019--kubernetes-io-main-staging.netlify.app/ja/docs/concepts/workloads/controllers/daemonset/#%E3%83%87%E3%83%95%E3%82%A9%E3%83%AB%E3%83%88%E3%82%B9%E3%82%B1%E3%82%B8%E3%83%A5%E3%83%BC%E3%83%A9%E3%83%BC%E3%81%AB%E3%82%88%E3%81%A3%E3%81%A6%E3%82%B9%E3%82%B1%E3%82%B8%E3%83%A5%E3%83%BC%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%95%E3%82%8C%E3%82%8B%E5%A0%B4%E5%90%88) said "FEATURE STATE: Kubernetes v1.23 [stable]", but code is set "1.17" as kubernetes version.

File: content/ja/docs/concepts/workloads/controllers/daemonset.md (hunk 1 above)
Comment (atoato88) [reply content/ja/docs/concepts/workloads/controllers/daemonset.md:83]: and you can find other place which using `for_k8s_version` property by command following:
`grep -r "for_k8s_version"`

Comment (atoato88) [reply content/ja/docs/concepts/workloads/controllers/daemonset.md:83]: Thank you to comment.

I created [PR which fix this topic, and merged](https://github.com/kubernetes/website/pull/33071).
Could you update this PR or close this and create other PR?

**Comment by Arhell** [comment]
@atoato88 done

**Comment by atoato88** [comment]
/lgtm

Thank you to update.

**Pull request merged**